### PR TITLE
Merge Master

### PR DESCRIPTION
**Pull request by nicard**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "robbiep/zbar-qrdecoder",
+    "name": "nicard/zbar-qrdecoder",
```
Comment (maestroprog) [inline composer.json:2]: Quote from packagist.org
> Do not submit forks of existing packages. If you need to test changes to a package that you forked to patch, use VCS Repositories instead. If however it is a real long-term fork you intend on maintaining feel free to submit it.

File: src/ZbarQrdecoderServiceProviderLaravel5.php [hunk 2]
```diff
@@ -6,38 +6,37 @@
...
-			return new ZbarDecoder($config, $processBuilder);
-		});
+    /**
+     * Register the service provider.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        //$this->app['zbardecoder'] = $this->app->share(function($app)
```
Comment (maestroprog) [inline src/ZbarQrdecoderServiceProviderLaravel5.php:26]: please, remove this comment line

**Review by maestroprog — CHANGES_REQUESTED** [review]
We need to decide what to do with the package name. robbip my PR https://github.com/robbiepaul/zbar-qrdecoder/pull/7 does not see. What are the future proposals?

**Comment by nicard** [comment]
Sorry for this commit, i don't will send requests anymore. It's a mistake.

**Comment by nicard** [comment]
Sorry for this commit, i don't will send requests anymore. It's a mistake.

**Comment by maestroprog** [comment]
Ok.

**Pull request closed without merging**